services/metadata/src/metadata/util/csv.clj

```diff
@@ -0,0 +1,8 @@
+(ns metadata.util.csv
+  (:require [clojure.data.csv :as csv]))
```

Comment:
I used clojure.data.csv rather than opencsv because it made more human-friendly CSV files -- quoting only when necessary, not for every field. Its interface is also nicer since it's clojure and not java.